libs/dh/market-participant/data-access-api/src/lib/dh-market-participant-edit-actor-data-access-api.store.ts

```diff
@@ -51,6 +51,7 @@ export interface MarketRoleChanges {
 export interface MarketRole {
   marketRole: EicFunction;
   gridAreas: MarketRoleGridArea[];
+  comment?: string | null;
```

Comment:
Is it necessary to have this property be optional?

Reply:
I did it so it would look like the auto generated one

```javascript
export interface ActorMarketRoleDto { 
    eicFunction: EicFunction;
    gridAreas: Array<ActorGridAreaDto>;
    comment?: string | null;
}
```

https://github.com/Energinet-DataHub/greenforce-frontend/blob/c13bca743aa3a3c7e00d80e7f1c34415924a667e/libs/dh/shared/domain/src/lib/generated/v1/model/actor-market-role-dto.ts#L23

But you are right it is kind of strange that it adds null and not just comment?: string .. :-)

Reply:
Fixed in https://github.com/Energinet-DataHub/greenforce-frontend/pull/1114/commits/4069d56e668a1cfeac17e111e7f1ab4d9d487175